examples/tensor/fixed_rank_prod_expressions.cpp

```diff
@@ -0,0 +1,169 @@
+//
+//  Copyright (c) 2018-2020, Cem Bassoy, cem.bassoy@gmail.com
+//  Copyright (c) 2019-2020, Amit Singh, amitsingh19975@gmail.com
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Google
+//
+
+
+#include <boost/numeric/ublas/tensor.hpp>
+#include <boost/numeric/ublas/matrix.hpp>
+#include <boost/numeric/ublas/vector.hpp>
+#include <iostream>
+
+int main()
+{
+    using namespace boost::numeric::ublas;
+
+    using format_t  = column_major;
+    using value_t   = float; // std::complex<double>;
+    using matrix_t = matrix<value_t,format_t>;
+    using vector_t = vector<value_t>;
+
+    // Tensor-Vector-Multiplications - Including Transposition
+    // dynamic_extents with static rank
+    {
+
+        auto n = dynamic_extents<3>{3,4,2};
```

Comment:
Recommend to place
```cpp
using extents_3_t = dynamic_extents<3>;
using extents_4_t = dynamic_extents<4>;
```
after line 27 and use those types.

Reply:
ok